README.md

```diff
@@ -191,7 +191,6 @@ The IOS procedure is similar to the macOS one:
 2. Update the submodules:
 ```
   $ git submodule init
-  $ git submodule update --remote
```

Comment:
This should be `git submodule update` (just missing the --remote). By running `git submodule update` after a pull a dev will bring their version up to that of the repo's.